brand new story

```diff
@@ -0,0 +1 @@
+It was 5 am in the morning. John Doe was looking at his steaming cup of coffee and was thinking about the previous 72 hours of his life.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.